source/Cosmos.Core_Plugs/System/TypeImpl.cs

```diff
@@ -24,16 +24,16 @@ public static Type GetTypeFromHandle(ulong aHandle)
             return xType;
         }
 
-        public static bool op_Equality(CosmosRuntimeType aLeft, CosmosRuntimeType aRight)
+        public static bool op_Equality(Type aLeft, Type aRight)
```

Comment:
Why is this type changing?

Reply:
Because that's the real signature, no idea why the other one was working, it wasn't supposed to I think.